cloudformation.yaml

```diff
@@ -0,0 +1,122 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Validates fulfilment information for a given subscription name
+Parameters:
+  App:
```

Comment:
this cf script is for  this app only I think? so maybe the app name can be in a constants block